### DISCUSSION TITLE
Question - Z1 Support

### DISCUSSION THREAD

**Discussion by ghost** [sn4k3/klipper-msla#2] [Q&A]
Hello sn4k3, amazing work on this fork!

We are designing an MSLA printer around the basis of your work and I had the above question before we develop more. Is Z1 for Z Tilt still supported as in base Klipper?

Many thanks in advance, no urgency!

**Comment by sn4k3** [comment]
Hello, 

The msla fork have no breaking changes nor restrictions. So you can use everything as before, including z_tilt (when you have more than one Z motor)
